index.d.ts

```diff
@@ -1,7 +1,7 @@
 export interface streamUploadInput {
   apiKey: string,
   apiSecret: string,
-  data: any,
+  stream: any,
```

Comment:
It is called stream here:
https://github.com/FleekHQ/fleek-storage-js/blob/30f779446913a4699aaf963dea0fc5fb7762c20f/methods/streamUpload.js#L9

Typescript was complaining. :)

Reply:
hmm on second thought it seems that other parts of the code expects it to be called data

Reply:
have you guys tested this code? its really buggy

Reply:
I get `AccessDenied` when it reaches the upload step of `s3-streaming-upload` but I've verified the credentials are correct.

Is it possible to just bypass the fleek sdk and make an axios request? If so, what needs to be in the headers?

Reply:
Unrelated, but I was trying to set the bucket name to something other than the default because I couldn't find a way to add new buckets in the website, and thought it might be created via the API. That resulted in Access Denied errors but no mention of the bucket. 

Is it not possible to add more buckets?